### PR TITLE
Linting rules and types enforce

### PR DESCRIPTION
**Pull request by sebbalex**
- using @pagopa/eslint-config linting rules, with some little exceptions
- code refactoring to enforce typings and linting
- some minor changes

### CODE REVIEW

**Comment by sebbalex** [comment]
@emilioSp these are the lint rules applied for typescript, wdyt?

File: src/schemaLoader.ts [hunk 1]
```diff
@@ -1,110 +1,117 @@
...
+          throw err;
+        }
+        if (schema === null) {
+          throw new Error(`${schema}`);
+        }
+
+        fetchSchema(uischemaURL, true)
+          .then((uischema) => {
+            const ui: UISchemaElement = JSON.parse(JSON.stringify(uischema));
+            const sc: JsonSchema = JSON.parse(JSON.stringify(schema));
```
Comment (sebbalex) [inline src/schemaLoader.ts:104]: I do not particularly love this.. but as of now is needed

**Review by emilioSp — APPROVED** [review]
LGTM
btw we will find trouble only using it...🚀🚀🚀

File: src/schemaLoader.ts (hunk 1 above)
Comment (emilioSp) [reply src/schemaLoader.ts:104]: The cost of typings 🙃

**Pull request merged**